src/common/header.css

```diff
@@ -0,0 +1,83 @@
+.header-container {
+    position: fixed;
+    top: 0;
+    background: white;
+    min-width: 100%;
+    border-bottom: 3px solid #F4F4F4;
+
+}
+
+header {
```

Comment:
See comment above: we should remove these styles for plain tags, or give them a specific class name